web_programming/fetch_amazon_product_data.py

```diff
@@ -42,7 +41,7 @@ def get_product_info(product: str = "laptop") -> None:
             "Discount",
         ]
     )  # initializing a pandas dataframe to store the requisite information
-    for i, j in it.zip_longest(
+    for i,j in it.zip_longest(
```

Comment:
[psf/black](https://github.com/psf) is going to undo this change in `pre-commit`

Reply:
So,isn't there any way to use the two simultaneously? Or should I remove the second variable?